src/Documentation/sidebar.json

```diff
@@ -65,22 +63,19 @@
         "label": "File Format (.dvc)",
         "slug": "dvc-file-format"
       },
+      "dvcignore",
       "large-dataset-optimization",
       "external-dependencies",
       {
         "label": "Managing External Data",
-        "slug": "external-outputs"
+        "slug": "external-cache-and-outputs"
```

Comment:
I would actually not change it now. Let's wait when we have more clarify on how do we want to structure this. Otherwise, why not `managing-external-data`?

Reply:
I don't really like **Managing External Data** actually. Why isn't **External Dependencies** included in that guide? I think **External Cache and Outputs**, which is what this article is about, should be not only the slug but the title (except that it's too long for the menu/nav bar). Maybe just **External Outputs** in the menu?

Reply:
it was before. Yes, it's confusing for now. The idea is to eventually (and I was waiting for the engine to properly support 3 levels) - is to have one single Manage External Data second level section, that will include External Outputs, External Dependencies, External Cache, Shared Cache (or a better title to explain cases like, https://github.com/iterative/dvc.org/pull/565 by @dashohoxha , https://github.com/iterative/dvc.org/pull/455), other related articles.

We'll have to go three levels to put more structure and scale growing UG.

Reply:
For now I've changed it to Manage External Data to simplify a new user experience - External Outputs do not mean anything to them, especially the name is extremely confusing when you use External _output_ to deal with _input_ (source, raw) data - this a very common case.

Reply:
Understood. I renamed it `managing-external-data` for now then (in 3d486bc).

I moved your other remarks to https://github.com/iterative/dvc.org/issues/654#issuecomment-536330090.

Reply:
👍